src/apps/Scripts/BaseScripts.js

```diff
@@ -66,6 +66,22 @@ const baseScripts = {
       # )
 
       # print (new_user)`
+  },
+  base_php: {
+    source: `
+      // Start coding!
+      // Note: You do not need opening / closing PHP tags in this script
+
+      $count = count($CONFIG);
+
+      foreach ($CONFIG as $key => $val) {
+        if (--$count < 0) {
+          break;
+        }
+
+        echo "$key = $val\n";
```

Comment:
Closing quote tag ends in new line
<img width="197" alt="screen shot 2016-10-21 at 17 29 27" src="https://cloud.githubusercontent.com/assets/11082340/19604062/f266e8e4-97b3-11e6-8b21-5bea87232d1e.png">